apps/webClient/src/components/View/index.tsx

```diff
@@ -41,48 +112,64 @@ const useStyles = makeStyles(theme => ({
     flexGrow: 1,
   },
   redPageWrapper: {
-    backgroundColor: RED,
-    background: RED_GRADIENT,
+    backgroundColor: DARK_ORANGE,
+    backgroundImage: RED_GRADIENT,
   },
   greenPageWrapper: {
-    backgroundColor: GREEN,
-    background: GREEN_GRADIENT,
+    backgroundColor: DARK_GREEN,
+    backgroundImage: GREEN_GRADIENT,
   },
   bluePageWrapper: {
-    backgroundColor: BLUE,
-    background: BLUE_GRADIENT,
+    backgroundColor: DARK_BLUE,
+    backgroundImage: BLUE_GRADIENT,
   },
   yellowPageWrapper: {
-    backgroundColor: YELLOW,
-    background: YELLOW_GRADIENT,
+    backgroundColor: DARK_YELLOW,
+    backgroundImage: YELLOW_GRADIENT,
+  },
+  "@keyframes homeToOurWork": {
+    "0%": { backgrounPosition: "93% 0%" },
+    "100%": { backgroundPosition: "0% 100%" },
+  },
+  homeToOurWork: {
+    backgroundImage: `linear-gradient(225deg, #f47920, #b12029, #8ec640, #0e7c3f)`,
+    backgroundSize: "800% 800%",
+    animation: "$homeToOurWork 7s ease",
   },
 }));
 
 const View: React.FC<Props> = ({ children }) => {
   const classes = useStyles();
   const { pathname } = useLocation();
+  const prevPath = usePrevious(pathname);
 
-  const derivePageRootBackgroundColor = (path: string) => {
-    let coloredPageRoot = [classes.root];
+  const derivePageRootBackgroundColor = (path: string, prevPath: string) => {
```

Comment:
we don't need to pass any values into this method, we can use them from the outer function scope.